docs/4-user-guide/5-component-actions.md

```diff
@@ -103,23 +103,44 @@ actions:
       - cmd: echo "before"
 ```
 
-## Action Configuration
+## Common Action Configuration Keys
 
-Within each of the `action` lists (`before`, `after`, `onSuccess`, and `onFailure`), the following action configurations are available:
+The following keys are common to all action configurations (wait or command):
 
-- `cmd` - (required) the command to run
+- `description` - a description of the action that will replace the text the users sees when the action is running, e.g. `description: "File to be created"` would show `Waiting for "File to be created"` instead of `Waiting for "touch test-create-before.txt"`
```

Comment:
```suggestion
- `description` - a description of the action that will replace the text the user sees when the action is running, e.g. `description: "File to be created"` would show `Waiting for "File to be created"` instead of `Waiting for "touch test-create-before.txt"`
```